sklearn/preprocessing/data.py

```diff
@@ -1948,6 +1948,11 @@ class QuantileTransformer(BaseEstimator, TransformerMixin):
     subsample : int, optional (default=1e5)
         Maximum number of samples used to estimate the quantiles.
 
+    noise_variance : None or float, optional (default=None)
```

Comment:
Have you checked this is useful in a predictive learning or clustering context? It seems like we're adding lots of features in version 1 without checking that they're the most useful solutions.